src/nl/surf/eduhub_rio_mapper/rio/aangeboden_opleiding.clj

```diff
@@ -66,25 +66,25 @@
    :toelichtingVereisteToestemming :explanationRequiredPermission})
 
 (defn- course-program-timeline-override-adapter
-  [{:keys [name description validFrom abbreviation link] :as _periode}
-   {:keys [acceleratedRoute deficiency foreignPartners jointPartnerCodes propaedeuticPhase
-           requirementsActivities studyChoiceCheck] :as _rio-consumer}]
-  (fn [pk]
-    (case pk
-      :begindatum validFrom
-      :buitenlandsePartner foreignPartners
-      :deficientie (rio/ooapi-mapping "deficientie" deficiency)
-      :eigenNaamAangebodenOpleiding (common/get-localized-value name ["nl-NL" "nl"])
-      :eigenNaamInternationaal (common/get-localized-value name)
-      :eigenNaamKort abbreviation
-      :eigenOmschrijving (common/get-localized-value description ["nl-NL" "nl"])
-      :eisenWerkzaamheden (rio/ooapi-mapping "eisenWerkzaamheden" requirementsActivities)
-      :internationaleNaamDuits (common/get-localized-value-exclusive name ["de"])
-      :propedeutischeFase (rio/ooapi-mapping "propedeutischeFase" propaedeuticPhase)
-      :samenwerkendeOnderwijsaanbiedercode jointPartnerCodes
-      :studiekeuzecheck (rio/ooapi-mapping "studiekeuzecheck" studyChoiceCheck)
-      :versneldTraject (rio/ooapi-mapping "versneldTraject" acceleratedRoute)
-      :website link)))
+  [{:keys [name description validFrom abbreviation link consumers] :as _periode}]
+  (let [{:keys [acceleratedRoute deficiency foreignPartners jointPartnerCodes propaedeuticPhase
+                requirementsActivities studyChoiceCheck]} (common/extract-rio-consumer consumers)]
```

Comment:
Zoals besproken: check of we consumers nu altijd uit overrides halen en of we ze dan niet ook moeten verplichten.

Reply:
Jelmer gesproken, ze mogen verplicht gesteld worden, dat gebeurt in PR 179.